libcudacxx/include/cuda/std/detail/libcxx/include/barrier

```diff
@@ -185,6 +187,21 @@ public:
             }
         return __old_phase;
     }
+    _LIBCUDACXX_NODISCARD_ATTRIBUTE _LIBCUDACXX_INLINE_VISIBILITY
```

Comment:
question: @miscco Should this be 

```
_LIBCUDACXX_NODISCARD_ATTRIBUTE inline _LIBCUDACXX_INLINE_VISIBILITY
```

That is what `.arrive` has (note the added inline)